core/primitives/res/runtime_configs

```diff
@@ -0,0 +1 @@
+../../../nearcore/res/runtime_configs
```

Comment:
Link might not work well on Windows :man_shrugging:

Reply:
Hm.. Can someone with Windows test this out. Previously we were referencing outside the package and while the local build worked, `cargo package` didn't bundle those artifacts. This symlink aids `cargo` in knowing to copy over those files when packaging, but I'm now unsure if the build works on Windows.

Reply:
cc @Longarithm 

I'd maybe try to keep it simple and just move files themselves here.

Reply:
Are we doing that to make `primitives-core` independent? If so, I agree with @matklad that `core/primitives/res/runtime_configs` is a better place for configs.

@miraclx could you move the files in this PR? If yes, could you also please split `nearcore/res/README.md` correspondingly?

Reply:
I tried that on Windows, and indeed it did not work well. I feel this primitives/runtime screams for being extracted into a separate crate since including all these json blobs does not feel right